### PR TITLE
Corrections

### LINKED ISSUE #32: Code validation - Corrections
Estimation : 10h

### PR DESCRIPTION
**Pull request by LFZDavid**

### CODE REVIEW

File: src/Service/AvatarFileUploader.php [hunk 1]
```diff
@@ -11,7 +11,7 @@ class AvatarFileUploader extends FileUploader
     public function deleteAvatarFile(Avatar $avatar)
     {
         $file = $this->publicDirectory . $avatar->getUrl();
-        if(file_exists($file)) {
+        if(file_exists($file) && $avatar->getUrl() != '/img/users/default.png') {
```
Comment (crazymad) [inline src/Service/AvatarFileUploader.php]: const ?

File: src/Service/FileUploader.php [hunk 2]
```diff
@@ -41,7 +41,7 @@ public function upload(UploadedFile $file)
     public function deleteFile(Media $media)
     {
         $file = $this->publicDirectory . $media->getUrl();
-        if(file_exists($file)) {
+        if(file_exists($file) && $media->getUrl() != '/img/tricks/default.jpg') {
```
Comment (crazymad) [inline src/Service/FileUploader.php]: const ?

File: src/Repository/TrickRepository.php [hunk 3]
```diff
@@ -19,28 +19,23 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Trick::class);
     }
 
-    // /**
```
Comment (crazymad) [inline src/Repository/TrickRepository.php:22]: you could delete all the commented code in your repo

**Review by crazymad — APPROVED** [review]

File: src/Service/AvatarFileUploader.php (hunk 1 above)
Comment (LFZDavid) [reply src/Service/AvatarFileUploader.php]: fixed in Corrections branch

File: src/Service/FileUploader.php (hunk 2 above)
Comment (LFZDavid) [reply src/Service/FileUploader.php]: fixed in Corrections branch

File: src/Repository/TrickRepository.php (hunk 3 above)
Comment (LFZDavid) [reply src/Repository/TrickRepository.php:22]: Fixed in Corrections branch

**Pull request merged**